src/main/python/camdkit/model.py

```diff
@@ -1345,9 +1345,9 @@ def make_json_schema() -> dict:
     }
 
 class LensCustom(ArrayParameter):
-  """Until the OpenLensIO model is finalised, this list provides custom
-  coefficients for a particular lens model e.g. undistortion, anamorphic
-  etc
+  """This list provides optional custom additonal coefficients for a
+  particular lens model. The meaning of which would require negotiation
+  between a particular producer and consumer. 
```

Comment:
...lens model, the meaning of...

I will raise this in the main slack, but given that people might have, e.g., custom accelerometer data, would this be better named `LensCustomDistortion`?

Reply:
It's not necessarily describing just distortion though - there could be other characteristics of the lens that a particular producer/consumer pair want to share... I'll clarify the comment...